src/index.html

```diff
@@ -131,22 +140,73 @@ <h2 id="faq">FAQ</h2>
 
     //Render icon-grid
     document.querySelector('.docs-icons').innerHTML = Object.keys(groups).map((type) => `
-      <h3 class="nrk-xs-12of12">${type}<br>&nbsp;</h3>
+      <h3 class="nrk-xs-12of12" style="color:#26292a">${type}<br>&nbsp;</h3>
```

Comment:
How about using template functions instead e.g.:

```
Object.keys(groups).map(templateType)

const templateType = (type) => {
`
  <h3 class="nrk-xs-12of12" style="color:#26292a">${type}<br>&nbsp;</h3>
  ${Object.keys(groups[type]).map(templateTypeName)
`
})

const templateTypeName = (name) => {
`
  <h3 class="nrk-xs-12of12" style="color:#26292a">${type}<br>&nbsp;</h3>
  ${Object.keys(groups[type]).map(templateTypeName)
`
})
```

and so on.
Easier to read.